Dockerfile

```diff
@@ -6,8 +6,8 @@ COPY . workdir/
 
 WORKDIR workdir
 
-RUN GRADLE_USER_HOME=cache ./gradlew -I gradle/init-publish.gradle buildDeb \
-       -I gradle/init-publish.gradle \
+RUN GRADLE_USER_HOME=cache ./gradlew -PenablePublishing=true buildDeb \
+       -PenablePublishing=true \
```

Comment:
I'm not sure why `-I gradle/init-publish.gradle` was there twice, but I think it should have only been there once and that it can just be replaced with a single `-PenablePublishing=true`

Reply:
Removed the second one!